deploy/standard/olm-catalog/kubevirt-hyperconverged/0.0.1/kubevirt-hyperconverged-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -5,19 +5,59 @@ metadata:
   namespace: placeholder
   annotations:
     capabilities: "Full Lifecycle"
-    categories: "Virtualization"
-    alm-examples: |
+    categories: "OpenShift Optional"
+    certified: "false"
+    support: "false"
+    containerImage: quay.io/community-operators/kubevirt/hyperconverged-cluster-operator:v2.0.0-4
+    repository: https://github.com/kubevirt/hyperconverged-cluster-operator
+    createdAt: 2019-04-17T16:01:59Z
+    alm-examples: >-
       [
         {
-          "apiVersion":"hco.kubevirt.io/v1alpha1",
-          "kind":"HyperConverged",
+          "apiVersion": "hco.kubevirt.io/v1alpha1",
+          "kind": "HyperConverged",
           "metadata": {
-            "name":"kubevirt-hyperconverged",
-            "namespace":"kubevirt-hyperconverged"
+            "name": "kubevirt-hyperconverged",
+            "namespace": "kubevirt-hyperconverged"
           },
           "spec": {
-            "CDIImagePullPolicy":"IfNotPresent",
-            "KubeVirtImagePullPolicy":"IfNotPresent"
+            "CDIImagePullPolicy": "IfNotPresent",
```

Comment:
Have the `spec` for each operator empty.  We don't want to expose anything here yet.